runtime/containerd/containerd.go

```diff
@@ -37,6 +39,8 @@ const (
 	defaultTimeout    = 30 * time.Second
 )
 
+var cniPath string
```

Comment:
let's not make this var global, it is not needed it seems

put it under `Init()` function where it is needed to check the cni bins

I'd do like this:

```go
cniBinPath := cniBin // init the path to be the default one
if cniBinEnvPath, ok = os.LookupEnv("CNI_BIN"); ok { // change the condition to be ok
		cniBinPath = cniBinEnvPath
	}
```

Reply:
and you don't need to declare `ok`, before that, just use `:=` in the `if` statement

Reply:
The path is used on cni execution as well... hence I've pulled that into init to not duplicate code that could then diverse ... 
then using the `:=` operator did not work ...
```
func cniInit(cId, ifName string, mgmtNet *types.MgmtNet) (*libcni.CNIConfig, *libcni.NetworkConfigList, *libcni.RuntimeConf, error) {
	// allow overwriting cni plugin binary path via ENV var

	cnic := libcni.NewCNIConfigWithCacheDir([]string{cniPath}, cniCache, nil)

	cniConfig := fmt.Sprintf(`
	{
		"cniVersion": "0.4.0",
		"name": "clabmgmt",
		"plugins": [
```